janaganana/settings.py

```diff
@@ -1,20 +1,41 @@
 # pull in the default wazimap settings
 from wazimap.settings import *  # noqa
+import os
 from decouple import config
 
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
 DEBUG = config('DJANGO_DEBUG', default=True, cast=bool)
 TEMPLATE_DEBUG = DEBUG
 
+SECRET_KEY = 'wt08^t@ugzs4sw*qn=c*=$d+jgkqkkp4$0z98j-k5s!o2um$(n'
+
 # DJANGO_SETTINGS_MODULE = config('DJANGO_SETTINGS_MODULE')
 
 # install this app before Wazimap
-INSTALLED_APPS = ['janaganana', 'django.contrib.sitemaps'] + INSTALLED_APPS
+INSTALLED_APPS = [
+    'django.contrib.admin',
```

Comment:
Just wondering why we should be hardcodinh insattledapps when we can directly import from wazimap and add it here as that is future proof.

Reply:
fixed